kubernetes/replicator-gke-cc/docs/index.rst

```diff
@@ -0,0 +1,510 @@
+
+#.  To view the ``stock-trades`` events on the destination (Confluent Cloud) cluster, in a second terminal open another shell on the ``client-console`` pod:
+
+    ::
+
+        kubectl -n operator exec -it client-console bash
+
+#.  Again, use the ``kafka-console-consumer`` to stream the values from the ``stock-trades`` topic, this time from the destination cluster.   The required configuration has been mounted in a volume on the pod in the ``/etc/destination-cluster-client-properties/destination-cluster-client.properties`` file.   Additionally, the bootstrap server value has been added in the ``/etc/destination-cluster-client-properties/destination-cluster-bootstrap`` file.  This command will use both of those files to create the connection to the destination cluster:
+
+    ::
+
+        kafka-console-consumer --bootstrap-server $(cat /etc/destination-cluster-client-properties/destination-cluster-bootstrap)  --consumer.config /etc/destination-cluster-client-properties/destination-cluster-client.properties --topic stock-trades --property print.value=false --property print.key=true --property print.timestamp=true
+
+    These commands will print out the timestamp and key of messages as they arrive in each cluster.  You an visually match up the events by these values and observe the replication process, for example, in terminal 1 you might see:
+
+    ::
+
+        ...
+        CreateTime:1572380698171        ZJZZT
+        CreateTime:1572380698280        ZWZZT
+        CreateTime:1572380698351        ZWZZT
+        CreateTime:1572380698577        ZJZZT
+        CreateTime:1572380699340        ZVZZT
+
+    And in terminal 2 shortly after identicial messages:
+
+    ::
+
+        ...
+        CreateTime:1572380698171        ZJZZT
+        CreateTime:1572380698280        ZWZZT
+        CreateTime:1572380698351        ZWZZT
+      
+Delete Resources
+~~~~~~~~~~~~~~~~
+
+After you are done evaluating the results of the demo, you can destroy all the provisioned Kubernetes resources with:
+
+.. sourcecode:: bash
+
+    make destroy-demo
+
+If you used the demo to create your cluster, you can destroy the GKE cluster with:
+
+.. sourcecode:: bash
+
+    make gke-destroy-cluster
+
+Highlights
+----------
+
+Deploying Kafka Connectors with Helm
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following highlights a simple Helm chart that can be used to deploy |kconnect-long| Connector configurations using the standard `Kafka Connect REST Interface <https://docs.confluent.io/current/connect/references/restapi.html>`__.  This is how this demonstration deploys the |crep-full| configuration, however, the same method could be used to deploy any |kconnect-long| configuration.  In future versions of |co-long|, |kconnect-long| connectors wil be managed by the Operator Controller `Controller <https://kubernetes.io/docs/concepts/architecture/controller/>`__.
```

Comment:
Add a reference to https://docs.confluent.io/current/tutorials/examples/kubernetes/gke-base/docs/index.html#connector-deployments

Reply:
I evolved the idea across these demos from standard K8s mainfests to a Helm chart.  I'm worried it could get confusing as they are comparable, but the Helm version here deploying replicator allows for better varability

Reply:
Gotcha.  Instead, how about calling out the datagen piece explicitly here to show how it's configured in this phase 2 part of the demo?

Reply:
I'll file a Jira on the potential improvements to the demo workflow, but the user looking for answers to deploying datagen connectors will need to find that elsewhere